src/core_modules/capture-core/utils/reactQueryHelpers/query/useMetadataQuery.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import { useQuery } from 'react-query';
+import type { QueryFunction, QueryKey } from 'react-query';
+import type { CustomOptions, Result } from './useMetadataQuery.types';
+
+const useAsyncMetadata = <TResultData>(
+    queryKey: QueryKey,
+    queryFn: QueryFunction<TResultData>,
+    { cacheTime, enabled }): Result<TResultData> => useQuery<TResultData>(queryKey, queryFn, {
+        enabled,
+        cacheTime,
+        staleTime: Infinity,
+    });
+
+export const useMetadataCustomQuery = <TResultData>(
+    queryKey: QueryKey,
+    queryFn: QueryFunction<TResultData>, {
+        cacheTime = 0,
+        enabled = true,
+    }: CustomOptions = {}): Result<TResultData> =>
+        useAsyncMetadata(queryKey, queryFn, { cacheTime, enabled });
+
+export const useIndexedDBQuery = <TResultData>(
```

Comment:
In this example, what is the difference between `useIndexedDBQuery` & `useMetadataCustomQuery`? What is the expected behavior here?